### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nikhil-zinjurde-imgtec**
Update Readme with post-repo reorganisation changes. Also, add packages after src-link changes to local packages in feeds.config. And, remove creator-docs as it has been removed from feeds

### CODE REVIEW

**Comment by abhijit-mahajani-imgtec** [comment]
Approved

File: README.md [hunk 1]
```diff
@@ -8,27 +8,28 @@ Manifest for cloning creator kit repositories.
...
 │   ├── openwrt
-│   ├── openwrt-img-feeds
+│   ├── openwrt-ckt-feeds
 |   |       ├──awalwm2m
 |   |       ├──button-gateway
 |   |       ├──device-manager
 |   |       ├──libflow-ex
 |   |       ├──libflow
 |   |       ├──webscripts
-│   └── openwrt-pkg-feeds
```
Comment (abhijit-mahajani-imgtec) [inline README.md]: cgilua
fping
should be added?

File: README.md [hunk 2]
```diff
@@ -8,27 +8,28 @@ Manifest for cloning creator kit repositories.
...
-├── docs
-│   └── creator_docs
+│   └── openwrt-feeds
```
Comment (abhijit-mahajani-imgtec) [inline README.md]: cgilua
fping
should be added?

Comment (nikhil-zinjurde-imgtec) [inline README.md]: Done

**Comment by nikhil-zinjurde-imgtec** [comment]
Approved

**Pull request merged**